tests/data/perl/report.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8'?>
+<testsuites>
+  <testsuite name="t/00_compile.t" errors="0" failures="0" skipped="0" tests="1" time="0.0876882076263428">
+    <system-out>ok 1 - use Example;
+1..1
+</system-out>
+    <testcase name="use Example;" classname="t/00_compile.t" time="0.0860052108764648" />
+  </testsuite>
+  <testsuite name="t/math/01_math.t" errors="0" failures="0" skipped="0" tests="3" time="0.0724310874938965">
+    <system-out>ok 1 - add(1, 2) == 3
+# Subtest: add
+    ok 1 - 1 + 2
+    ok 2 - 3 + 4
+    1..2
+ok 2 - add
+# Subtest: double
+    ok 1 - 1 + 2
+    ok 2 - 3 + 2
+    1..2
+ok 3 - double
+1..3
+</system-out>
+    <testcase name="add(1, 2) == 3" classname="t/math/01_math.t" time="0.0693521499633789" />
+    <testcase name="add" classname="t/math/01_math.t" time="0.000168085098266602" />
+    <testcase name="double" classname="t/math/01_math.t" time="0.000128984451293945" />
+  </testsuite>
+  <testsuite name="t/easy/01_easy.t" errors="0" failures="0" skipped="0" tests="2" time="0.154859066009521">
+    <system-out>ok 1 - a + b = ab
+ok 2 - str_concat {
+    ok 1 - a + dddd
+    ok 2 - bb + ccc
+    1..2
+}
+1..2
+</system-out>
+    <testcase name="a + b = ab" classname="t/easy/01_easy.t" time="0.152199029922485" />
+    <testcase name="str_concat {" classname="t/easy/01_easy.t" time="0.000449180603027344" />
+  </testsuite>
+</testsuites>
```

Comment:
How about adding a failure case, just in case?

Reply:
Added.
https://github.com/launchableinc/cli/pull/541/commits/33a9f9b29701794d88b8900f7dbda964ae415392